lib/poxa/supervisor.ex

```diff
@@ -15,8 +15,9 @@ defmodule Poxa.Supervisor do
     subscription_worker = worker(Watcher, [Poxa.Event, Poxa.SubscriptionHandler, []], [id: Poxa.SubscriptionHandler])
     children = [event_worker, subscription_worker]
     if web_hook do
-      web_wook_worker = worker(Watcher, [Poxa.Event, Poxa.WebHook, []], [id: Poxa.WebHook])
-      children = children ++ [web_wook_worker]
+      web_wook_watcher = worker(Watcher, [Poxa.Event, Poxa.WebHook.Handler, []], [id: Poxa.WebHook.Handler])
+      web_wook_dispatcher = worker(Poxa.WebHook.Dispatcher, [])
+      children = children ++ [web_wook_watcher, web_wook_dispatcher]
```

Comment:
I wonder if we should have a supervisor for these 2 workers and make them restart all together if one of them die (one for all)